documentation/book/proc-bridge-retrieving-latest-messages-from-consumer.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-quickstart.adoc
+
+[id='proc-bridge-retrieving-latest-messages-from-consumer-{context}']
+= Retrieving the latest messages from a Kafka Bridge consumer
+
+Next, retrieve the latest messages from a Kafka Bridge consumer by requesting data from the link:https://strimzi.io/docs/bridge/latest/#_poll[records^] endpoint. For production use, HTTP clients can retrieve messages repeatedly or as a one-off operation. 
+
+.Prerequisites
+
+* The Kafka Bridge is deployed to your Kubernetes cluster.
```

Comment:
I wonder about repeating prereqs in a quickstart, because we do expect the user to go through the guide in a linear way.

Reply:
Agreed! These are not really standlone procedures, so the prerequisites are cumbersome.